### PR TITLE
housekeeping for picker-network

### LINKED ISSUE #16640: 🧹 [UI House Keeping] `PickerNetwork`
### Description
Ensure that `PickerNetwork` adheres to all of the following conventions and standards

- [ ] Has a `className` prop and the PropType descriptions are all the same
- [ ] Prop table in MDX docs have the "Accepts all Box component props" description and link
- [ ] We are consistent when using the same prop names like `size` and are suggesting the use of the generalized `design-system.js` constants e.g. `SIZES` as the primary option but noting the component consts in the documentation and using them for propType validation and storybook controls only
- [ ] Standardize all similar prop names for images `imgSrc`, `imgAlt`(html element + attribute) (needs audit)
- [ ] We have a story for each component prop and we use the prop name verbatim e.g. `size` prop would be `export const Size = (args) => (`
- [ ] We have the accompanying documentation for each component prop and we use the prop name verbatim e.g. `size` prop would be `### Size`
- [ ] Are multiple props stories allowed? e.g. `Color, Background Color And Border Color` story in `base-avatar` - [ ] yes when it makes sense to
- [ ] All Base components follow the suffix convention e.g. `ButtonBase`
- [ ] All Base component MDX documentation have the base component notification at the top
- [ ] Add `mm-` prefix to all classNames
- [ ] className is kebab case version of the component name
- [ ] Spread base components props and reduce duplication of props when props aren't being changed and remain the same for both variant and base components
- [ ] Add component to root `index.js` file in component-library
- [ ] Add locals for any default text I18nContext as default context 
- [ ] Add any "to dos" with a `// TODO:` comment so we can search for them at a later date e.g. blocking components etc
- [ ] Add snapshot testing
- [ ] Add pixel values to propType descriptions if we use abstracted prop types that relate to pixel values e.g. `SIZE.MD (32px)`
- [ ] Each prop section in the MDX docs should have: a heading, a description, a story and an example code snipped

### PR DESCRIPTION
**Pull request by NidhiKJha**
* Fixes #16640 

This PR is to ensure that `PickerNetwork` adheres to all of the following conventions and standards

- [x] Has a `className` prop and the PropType descriptions are all the same
- [x] Prop table in MDX docs have the "Accepts all Box component props" description and link
- [x] We are consistent when using the same prop names like `size` and are suggesting the use of the generalized `design-system.js` constants e.g. `SIZES` as the primary option but noting the component consts in the documentation and using them for propType validation and storybook controls only
- [x] Standardize all similar prop names for images `imgSrc`, `imgAlt`(html element + attribute) (needs audit)
- [x] We have a story for each component prop and we use the prop name verbatim e.g. `size` prop would be `export const Size = (args) => (`
- [x] We have the accompanying documentation for each component prop and we use the prop name verbatim e.g. `size` prop would be `### Size`
- [ ] Are multiple props stories allowed? e.g. `Color, Background Color And Border Color` story in `base-avatar` - [ ] yes when it makes sense to
- [ ] All Base components follow the suffix convention e.g. `ButtonBase`
- [x] All Base component MDX documentation have the base component notification at the top
- [x] Add `mm-` prefix to all classNames
- [x] className is kebab case version of the component name
- x ] Spread base components props and reduce duplication of props when props aren't being changed and remain the same for both variant and base components
- [x] Add component to root `index.js` file in component-library
- [ ] Add locals for any default text I18nContext as default context 
- [ ] Add any "to dos" with a `// TODO:` comment so we can search for them at a later date e.g. blocking components etc
- [x] Add snapshot testing
- [ ] Add pixel values to propType descriptions if we use abstracted prop types that relate to pixel values e.g. `SIZE.MD (32px)`
- [x] Each prop section in the MDX docs should have: a heading, a description, a story and an example code snipped

### CODE REVIEW

File: ui/components/component-library/picker-network/README.mdx [hunk 1]
```diff
@@ -16,32 +16,32 @@ The `PickerNetwork` accepts all props below as well as all [Box](/docs/ui-compon
...
 
 ```jsx
-import { PickerNetwork } from '../../ui/component-library/picker-network';
-  <PickerNetwork src="./images/arbitrum.svg" />
-  <PickerNetwork src="./images/matic-token.png" />
-  <PickerNetwork src="./images/optimism.svg" />
+import { PickerNetwork } from '../../ui/component-library';
+  <PickerNetwork label="Arbitrum One" />
+  <PickerNetwork label="Polygon Mainnet" />
+  <PickerNetwork label="Optimism" />
```
Comment (georgewrmarshall) [inline ui/components/component-library/picker-network/README.mdx:31]: nit: formatting
```suggestion
import { PickerNetwork } from '../../ui/component-library';

<PickerNetwork label="Arbitrum One" />
<PickerNetwork label="Polygon Mainnet" />
<PickerNetwork label="Optimism" />
```

File: ui/components/component-library/picker-network/README.mdx [hunk 2]
```diff
@@ -16,32 +16,32 @@ The `PickerNetwork` accepts all props below as well as all [Box](/docs/ui-compon
...
 
 ```jsx
-import { PickerNetwork } from '../../ui/component-library/picker-network';
-  <PickerNetwork label="Arbitrum One" />
-  <PickerNetwork label="Polygon Mainnet" />
-  <PickerNetwork label="Optimism" />
+import { PickerNetwork } from '../../ui/component-library';
+  <PickerNetwork src="./images/arbitrum.svg" />
+  <PickerNetwork src="./images/matic-token.png" />
+  <PickerNetwork src="./images/optimism.svg" />
```
Comment (georgewrmarshall) [inline ui/components/component-library/picker-network/README.mdx:46]: nit: formatting
```suggestion
import { PickerNetwork } from '../../ui/component-library';

<PickerNetwork src="./images/arbitrum.svg" />
<PickerNetwork src="./images/matic-token.png" />
<PickerNetwork src="./images/optimism.svg" />
```

File: ui/components/component-library/picker-network/README.mdx [hunk 3]
```diff
@@ -16,32 +16,32 @@ The `PickerNetwork` accepts all props below as well as all [Box](/docs/ui-compon
...
+  <PickerNetwork label="Arbitrum One" />
+  <PickerNetwork label="Polygon Mainnet" />
+  <PickerNetwork label="Optimism" />
 ```
 
-### Label
+### Src
 
-Use the `label` prop for the text content of the `PickerNetwork` component
+Use the `src` prop with an image url to render the `AvatarNetwork`.
```
Comment (georgewrmarshall) [inline ui/components/component-library/picker-network/README.mdx:36]: Could add how to access the AvatarNetwork component
```suggestion
Use the `src` prop with an image url to render the `AvatarNetwork`. Use the `avatarNetworkProps` to pass additional props to the `AvatarNetwork` component.
```

**Review by georgewrmarshall — COMMENTED** [review]
Looking good. Couple nits and left this comment in slack channel avatarNetworkProps https://consensys.slack.com/archives/C02VD8QG0LT/p1671587335207129

**Review by garrettbear — APPROVED** [review]

**Review by georgewrmarshall — APPROVED** [review]

**Pull request merged**